tests/test_run_inputs.py

```diff
@@ -0,0 +1,129 @@
+from typing import Optional
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import (
+    ElementNotVisibleException,
+    ElementNotSelectableException,
+    NoSuchElementException,
+)
+from selenium.webdriver import ActionChains
+
+web = "https://machinist.flapflap.io/"
```

Comment:
It would be best to place this in a single location where it is globally accessible by all tests. This way we can configure the tests to target a local instance of the app too.

There is a file at the root of the repo called `options.py`. Can you place this constant there? Also, use constant naming convention: `HOMEPAGE` instead of `web`